asmcomp/cmm.ml

```diff
@@ -166,7 +166,8 @@ type expression =
   | Cifthenelse of expression * expression * expression
   | Cswitch of expression * int array * expression array * Debuginfo.t
   | Cloop of expression
-  | Ccatch of rec_flag * (int * (Ident.t * machtype) list * expression) list * expression
+  | Ccatch of rec_flag * (int * (Ident.t * machtype) list * expression) list *
+              expression
```

Comment:
When you break line in the middle variant constructor descriptions, it is nicer to have the star symbol at the beginning of the next line. 

This is my personal taste (in general with infix operators) but also a style that is used in the compiler:

```
$ git grep -B2 "^ *\*[^)][^o]" typing/*.mli | cat
typing/includemod.mli-  | Value_descriptions of Ident.t * value_description * value_description
typing/includemod.mli-  | Type_declarations of Ident.t * type_declaration
typing/includemod.mli:        * type_declaration * Includecore.type_mismatch list
--
typing/typedtree.mli-  | Tcl_fun of
typing/typedtree.mli-      arg_label * pattern * (Ident.t * expression) list
typing/typedtree.mli:      * class_expr * partial
```

Reply:
Ah, OK - I thought I found conflicting examples (or maybe I couldn't, and just went along with arrows, which I think are often left dangling).

Reply:
Hm, looking for it (`git grep -B2 "[^(]\* *$" *.mli`) there are examples of using the opposite style all over, some of which written by myself... Then there is this nice grep result:

```
typing/typedtree.mli      * class_expr * partial
typing/typedtree.mli  | Tcl_apply of class_expr * (arg_label * expression option) list
typing/typedtree.mli  | Tcl_let of rec_flag * value_binding list *
```

In other words: do as you please.

Reply:
I suppose another option for these particular long tuple types might be to go down the same road as for records:

```ocaml
 | Ccatch of rec_flag
           * (int * (Ident.t * machtype) list * expression) list
           * expression
```

Reply:
(although aligning the asterisks with the `rec_flag` is probably better ... and possibly even admitting a redundant first `*` as we do for `|`?!?!)